src/components/SyncContainer.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import withRouter from 'react-router-dom/withRouter';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { locationChange } from '../actions/syncActionCreators';
+
+function mapDispatchToProps(dispatch) {
+	return {
+		dispatchLocationChange: bindActionCreators(locationChange, dispatch),
+	};
+}
+
+/**
+ * @module SyncContainer
+ */
+export class SyncContainer extends React.Component {
+	/**
+	 * This container only cares about routing changes. When the route changes,
```

Comment:
I know you're describing it here, but I'm still unclear as to what this container does, and possibly also why it's a separate component as opposed to functionality with `RouteLayout`